modules/dedicated-scaling-your-cluster.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * getting_started/scaling_your_cluster.adoc
+
+[id="dedicated-scaling-your-cluster_{context}"]
+= Scaling your cluster
+
+To scale your {product-title} cluster:
+
+. From link:https://cloud.redhat.com/openshift[cloud.redhat.com/openshift], click
+ on the cluster you wish to resize.
+
+. Click the *Actions* button, then *Scale Cluster*.
+
+. Select how many compute nodes are required, then click *Apply*.
+
+Scaling will occur automatically. In the *Overview* tab under the *Details*
```

Comment:
s/will occur/occurs

Reply:
In the *Overview* tab under the *Details* heading, the *Status* indicator shows that your cluster is *Ready* for use.